File: Data Structures & Algorithms/concatenation-of-array/submission-11.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] nums_out = new int[n * 2];
        
        int j = 0;

        for(int i = 0; i < n * 2; i++) {    
            if (j == n) {
                j = 0;
            }

            nums_out[i] = nums[j];
            j++;
        }

        return nums_out;
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-11.cs
public class Solution {
    public int[] TwoSum(int[] numbers, int target) {
        int left = 0; int right = numbers.Length;

        for(int i = 0; i < numbers.Length; i++) {
            int sum = nums[left] + nums[right];

            if (sum == target) {
                return {left + 1, right + 1};
            } else if (sum > target) {
                right--;
            } else {
                left++;
            }

        }

        return new int[0];
    }
}
